app/views/layouts/admin.html.erb

```diff
@@ -30,28 +29,33 @@
   data-ga-tracking="<%= Settings::General.ga_tracking_id %>"
   data-controller="snackbar"
   data-action="snackbar:add@document->snackbar#addItem">
+  <div id="body-styles">
+    <style>
+      :root {
+        --accent-brand-lighter-rgb: <%= Color::CompareHex.new([Settings::UserExperience.primary_brand_color_hex]).brightness(1.35, only_values: true) %>;
+        --accent-brand-rgb: <%= Color::CompareHex.new([Settings::UserExperience.primary_brand_color_hex]).brightness(1, only_values: true) %>;
+        --accent-brand-darker-rgb: <%= Color::CompareHex.new([Settings::UserExperience.primary_brand_color_hex]).brightness(0.8, only_values: true) %>;
+      }
+    </style>
+  </div>
   <header class="crayons-header">
     <span id="route-change-target" tabindex="-1"></span>
     <a href="#main-content" class="skip-content-link"><%= t("views.main.header.skip") %></a>
     <div class="crayons-header__container">
       <%= render "layouts/logo" %>
 
-      <h1 class="pl-4 fs-l">
-        <a href="<%= admin_path %>">Admin</a>
-        <span class="color-base-60">&raquo;</span> <span class="fw-bold">
+      <h1 class="pl-4 fs-base hidden s:block">
+        <a class="c-link c-link--branded" href="<%= admin_path %>">Admin</a>
+        <span class="color-base-60">&raquo;</span>
+        <span class="fw-medium">
           <%= yield(:h1).presence || controller_name.titleize %>
         </span>
       </h1>
 
-      <div class="flex items-center h-100 ml-auto">
-        <a href="https://admin.forem.com/" class="crayons-btn crayons-btn--ghost-brand hidden mr-2 whitespace-nowrap s:block">
-          Forem Admin Guide
-        </a>
-
-        <a href="/" class="crayons-btn hidden mr-2 whitespace-nowrap s:block">
-          Go to <%= community_name %> home page
-        </a>
-      </div>
+      <nav class="flex items-center h-100 ml-auto">
```

Comment:
```suggestion
      <nav aria-label="header" class="flex items-center h-100 ml-auto">
```

now we have multiple `nav`s on the page, labelling them has a big impact on screen reader users being able to get to the right one easily. TBH I couldn't think of a better name than "header" which would be announced as "header navigation", but happy for it to be called something else

Reply:
thank you!

i also can't think of a better name 🤷